src/styles/utils/animation.scss

```diff
@@ -0,0 +1,19 @@
+@keyframes moveSecond {
+  0% {
+    @include animation-arrows-to;
+  }
+
+  100% {
+    @include animation-arrows-from;
+  }
+}
+
+@keyframes moveMinute {
+  0% {
+    @include animation-arrows-to;
+  }
+
+  100% {
+    @include animation-arrows-from;
+  }
+}
```

Comment:
that's totally same keyframes, you don't need to copy yourself

Reply:
also there's no need to create different files in this task

Reply:
Now is Done.